observability/deploy/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dummy
```

Comment:
Should this be `observability` given what you mentioned in the development docs? I assume this is the namespace that will be deployed to host services.

Reply:
Its actually ignored / replaced with whatever is configured in the [kustomize](https://github.com/Azure/ARO-HCP/pull/1319/files#diff-8e8586d5d7525f12455211f051dc350e8bd960df88a39dfa7f69ef893c7c11a1R4) file.

Reply:
Then, maybe use "placeholder" for that? That's the convention.